erfa/_dev/scm_version.py

```diff
@@ -6,43 +6,55 @@
 import os.path as pth
 from warnings import warn
 
-try:
-    from setuptools_scm import git, Configuration, get_version as _get_version
```

Comment:
I'm a bit confused why this is needed, given that astropy has the simpler direct `try/except` that we had here. What other attribute is being gotten that the `__getattr__` solves? Or is it just that timing now is different?

Reply:
I didn't have any recollection of it, luckilly the answer to your question is in the commit message:
`RFC: refactor scm_version.py to survive pytest collection if setuptools_scm isn't installed`

Reply:
So, isn't the answer then to ensure that pytest doesn't look at this directory? Is that what astropy does?

Reply:
I don't know. I couldn't reproduce the problem I was trying to solve here so I just reverted the change. The PR is more self-contained as a result.